mkdocs.yml

```diff
@@ -48,6 +48,11 @@ markdown_extensions:
   - pymdownx.emoji:
       emoji_index: !!python/name:material.extensions.emoji.twemoji
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
+  - pymdownx.superfences:
```

Comment:
please add the custom fences to the superfences entry above to avoid duplication